code/game/objects/structures/traps.dm

```diff
@@ -81,10 +85,79 @@
 	name = "shock trap"
 	desc = "A trap that will shock and render you immobile. You'd better avoid it."
 	icon_state = "trap-shock"
+	var/stun_time = 100
 
 /obj/structure/trap/stun/trap_effect(mob/living/L)
 	L.electrocute_act(30, src, safety=1) // electrocute act does a message.
-	L.Paralyze(100)
+	L.Paralyze(stun_time)
+
+/obj/structure/trap/stun/hunter
```

Comment:
Don't these things have anitmagic checks and such wizard stuff.

Reply:
done